### DISCUSSION TITLE
arm64跑起来了（喜

### DISCUSSION THREAD

**Discussion by Miuzarte** [Tsuk1ko/CQ-picfinder-robot#450] [一般讨论]
nvm: 
```shell
wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
```
nodejs: 
```shell
nvm install 18.19.0
```
~~config.canvasLibrary需要用canvas~~
```jsonc
    // Canvas 库选择，可选值 ["auto", "@napi-rs/canvas", "canvas"] 详见 wiki
    "canvasLibrary": "auto",
```
tls用1.2

![image](https://github.com/Tsuk1ko/cq-picsearcher-bot/assets/66856838/b4be7d94-7358-40e1-a2a1-437e562c0e9d)

**Comment by Tsuk1ko** [comment]
按理来说你依赖装成功了就说明应该可以用 @napi-rs/canvas

musl 那个失败了是没关系的，gnu 的装成功了就可以了

<img width="644" alt="image" src="https://github.com/Tsuk1ko/cq-picsearcher-bot/assets/24877906/57069f63-1204-4288-bfba-6f82754d0efd">
